src/websiteAccount/WebsiteAccount.java

```diff
@@ -124,46 +132,73 @@ public static boolean addWebsiteAccountPrompts(ConnectionSource databaseConnecti
 		return websiteAccount.addWebsiteAccount(databaseConnection);
 	}
 	
-	public static String viewWebsiteAccountLogin(ConnectionSource databaseConnection, Scanner keyboard, User safeStoreUser) {
+	/**
+	 * gets the foreign collection of all website account rows for that user
+	 * @param databaseConnection
+	 * @param safeStoreUser of type User
+	 * @return a ForeignCollection<WebsiteAccountEntity> that holds all the db rows of accounts
+	 * 			for the safeStoreUser
+	 */
+	public static ForeignCollection<WebsiteAccountEntity> getAllWebsiteAccounts(ConnectionSource databaseConnection, User safeStoreUser) {
+		Dao<UserEntity, String> userDao;
 		try {
-			// get user object from db to view accounts
-			Dao<UserEntity, String> userDao = DaoManager.createDao(databaseConnection, UserEntity.class);
-			UserEntity returnedFromDb = userDao.queryForSameId(safeStoreUser.getUserEntity());
-			ForeignCollection<WebsiteAccountEntity> websiteAccounts = returnedFromDb.getWebsiteAccounts();
-			
-			// print out all the accounts for the user
-			if (websiteAccounts.isEmpty()) {
-				System.out.println("No accounts");
-				return "No accounts";
-			}
+			userDao = DaoManager.createDao(databaseConnection, UserEntity.class);
+			UserEntity userEntity = userDao.queryForSameId(safeStoreUser.getUserEntity());
+			return userEntity.getWebsiteAccounts();
+		} catch (SQLException e) {
+			e.printStackTrace();
+			return null;
+		}
+	}
+	
+	/**
+	 * Prints a list of all website accounts for the User
+	 * @param databaseConnection
+	 * @param safeStoreUser of type User
+	 */
+	public static void printAllWebsiteAccounts(ConnectionSource databaseConnection, User safeStoreUser) {
+		ForeignCollection<WebsiteAccountEntity> websiteAccounts = getAllWebsiteAccounts(databaseConnection, safeStoreUser);
+		if (websiteAccounts.isEmpty() || websiteAccounts == null) {
+			System.out.println("No accounts");
+		}
+		for (WebsiteAccountEntity account : websiteAccounts) {
+			System.out.println(Encryption.decrypt(account.getNickname()));
+		}
+	}
+	
+	/**
+	 * Prompts the user to pick the website account they want to view the login and password info for and prints it out
+	 * @param databaseConnection
+	 * @param keyboard
+	 * @param safeStoreUser
+	 * @return - what was printed for that specific website account...mainly used for testing
+	 */
+	public static String viewWebsiteAccountInfo(ConnectionSource databaseConnection, Scanner keyboard, User safeStoreUser) {
+		printAllWebsiteAccounts(databaseConnection, safeStoreUser);
+		ForeignCollection<WebsiteAccountEntity> websiteAccounts = getAllWebsiteAccounts(databaseConnection, safeStoreUser);
```

Comment:
i dont know if its too late, but like why do you get all website accounts? Seems like a lot of work each time we want to view information on a website account? Shouldn't we just query for the specific one they ask for?